api/tests/acceptance/application/certification-centers/certification-center-controller-get-import-template_test.js

```diff
@@ -9,26 +9,68 @@ describe('Acceptance | Controller | certification-center-controller-get-import-t
   });
 
   describe('GET /api/certification-centers/{certificationCenterId}/import', function () {
-    context('when user requests sessions import template', function () {
-      it('should return a csv file', async function () {
-        // given
-        const userId = databaseBuilder.factory.buildUser().id;
-        const certificationCenterId = databaseBuilder.factory.buildCertificationCenter().id;
-        databaseBuilder.factory.buildCertificationCenterMembership({ userId, certificationCenterId });
-        await databaseBuilder.commit();
-
-        // when
-        const response = await server.inject({
-          method: 'GET',
-          url: `/api/certification-centers/${certificationCenterId}/import`,
-          payload: {},
-          headers: { authorization: generateValidRequestAuthorizationHeader(userId) },
+    describe('when user requests sessions import template', function () {
+      context('when the certification center is not habilitated to any complementary certification', function () {
+        it('should return a csv file with no complementary certification column', async function () {
+          // given
+          const userId = databaseBuilder.factory.buildUser().id;
+          const certificationCenterId = databaseBuilder.factory.buildCertificationCenter().id;
+          databaseBuilder.factory.buildCertificationCenterMembership({ userId, certificationCenterId });
+          await databaseBuilder.commit();
+
+          // when
+          const response = await server.inject({
+            method: 'GET',
+            url: `/api/certification-centers/${certificationCenterId}/import`,
+            payload: {},
+            headers: { authorization: generateValidRequestAuthorizationHeader(userId) },
+          });
+
+          // then
+          expect(response.statusCode).to.equal(200);
+          expect(response.headers['content-type']).to.equal('text/csv; charset=utf-8');
+          expect(response.headers['content-disposition']).to.include('filename=import-sessions');
```

Comment:
On ne teste pas les headers comme l'annonce le test: 'should return a csv file with no complementary certification column'